### ISSUE TITLE
[feature request] Dock with higher resolution than 64x64 for Dock apps

### ISSUE THREAD

**Issue by ThomasHeckert** [window-maker/wmaker#47]
Hello,

I am new to wmaker so sorry if this is allways realized:

Is there a way to increase the prehistoric resolution of the Icons from  max 64x64 to at least 256x256 or more?

I would welcome this very much not only for Dock apps.

Best regards

Thomas

**Comment by gryf** [comment]
Yes, you can. Go to WPrefs->Icon Preferences and set icon size. Be warned though, most (all?) dockapps are written for 64x64 resolution, so they will stick at the center of the tile unscaled.
